packages/client/hmi-client/src/components/workflow/ops/model-config/model-config-operation.ts

```diff
@@ -14,6 +15,17 @@ export interface ModelConfigOperationState extends BaseState {
 	hasCodeBeenRun: boolean;
 }
 
+export const blankModelConfig: ModelConfiguration = {
+	id: '',
+	modelId: '',
+	name: '',
+	description: '',
+	calibrationRunId: '',
+	observableSemanticList: [],
+	parameterSemanticList: [],
+	initialSemanticList: []
+};
```

Comment:
We are actually using `ModelConfiguration.simulationId` and not the "calibrationRunId"